packages/@winglibs/testfixture/API.md

```diff
@@ -16,7 +16,9 @@
 
 ### PublicClass (preflight class) <a class="wing-docs-anchor" id="@winglibs/testfixture.PublicClass"></a>
 
-*No description*
+Shared behavior between all Wing SDK resources.
```

Comment:
Ha, I guess that's a little bit of an unintended change. Let me see if I can tune this a bit so we don't show this